sdk/storage/azure-storage-blob/swagger/src/main/java/BlobStorageCustomization.java

```diff
@@ -23,6 +23,11 @@ public void customize(LibraryCustomization customization, Logger logger) {
         PackageCustomization models = customization.getPackage("com.azure.storage.blob.models");
         models.getClass("PageList").addAnnotation("@JsonDeserialize(using = PageListDeserializer.class)");
 
+        models.getClass("BlobCopySourceTags").rename("BlobCopySourceTagsMode");
+        models.getClass("BlobErrorCode").getProperty("SNAPHOT_OPERATION_RATE_EXCEEDED")
```

Comment:
Do we want to keep this knowing that the method will throw based on testing it? Maybe comment it out for now until it is fix?

Reply:
Removed